projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/complex/read-complex-field.component.ts

```diff
@@ -30,8 +30,8 @@ export class ReadComplexFieldComponent extends AbstractFieldReadComponent implem
           field.list_items = this.caseField.value[field.id].list_items;
           field.value = {
             list_items: field.list_items,
-            value: this.caseField.value[field.id].value && this.caseField.value[field.id].value.code ?
-                    this.caseField.value[field.id].value.code :
+            value: this.caseField.value[field.id].value && this.caseField.value[field.id].value?.code ?
```

Comment:
@andywilkinshmcts - if you're doing `this.caseField.value[field.id].value?`, then the previous `this.caseField.value[field.id].value` is redundant